.github/workflows/docker-build.yml

```diff
@@ -204,27 +208,10 @@ jobs:
           just deploy cluster
           kubectl create secret generic regcred --from-file=.dockerconfigjson=$HOME/.docker/config.json --type=kubernetes.io/dockerconfigjson
           echo -e "\n\nDeploying with astria images tagged $TAG"
-          just deploy ibc-test-infra $TAG
-          just build-and-load-bridge-tester-image $TAG
-      - name: Run IBC Bridge Tester
-        timeout-minutes: 2
-        run: |
-          TAG=sha-$(git rev-parse --short HEAD)
-          printlogs() {
-            echo "IBC Transfer Test failed, printing logs..."
-            kubectl describe job bridge-tester-chart -n astria-dev-cluster
-            kubectl logs job/bridge-tester-chart --all-containers -n astria-dev-cluster
-            exit 1
-          }
-          just deploy bridge-tester $TAG
-          # timeout before the gh job times out so we can print logs
-          kubectl wait --for=condition=complete --timeout=90s job/bridge-tester-chart -n astria-dev-cluster || printlogs
-          JOB_STATUS=$(kubectl get job bridge-tester-chart -n astria-dev-cluster -o jsonpath='{.status.succeeded}')
-          if [ "$JOB_STATUS" != "1" ]; then
-            printlogs
-          else
-            echo "IBC Transfer Test passed"
-          fi
+          just ibc-test deploy $TAG
+      - name: Run IBC ICS20 Transfer test
+        timeout-minutes: 3
+        run: just ibc-test run ./celestia ./celestia-appd
```

Comment:
Similar to above: the arguments are required because we need to provide a celestia "home" directory, and the path to the celestia-appd executable. It would be neater if we could run straight from the container running celestia-appd.